WordPressEditor/WordPressEditor/Classes/Plugins/WordPressPlugin/Gutenberg/GutenbergOutputHTMLTreeProcessor.swift

```diff
@@ -129,21 +129,24 @@ private extension GutenbergOutputHTMLTreeProcessor {
     // MARK: - Gutenberg HTML Attribute Data
     
     private func gutenblockCloserData(for element: ElementNode) -> String? {
-        return decodedAttribute(named: GutenbergAttributeNames.blockCloser, from: element)
+        let decoder = GutenbergAttributeDecoder()
```

Comment:
We could make the decoder be a class property to avoid recreating it in each call to these methods.